### ISSUE TITLE
Rule: no-trailing-spaces in comments

### ISSUE THREAD

**Issue by asapach** [eslint/eslint#1229]
When trailing spaces are present in comments (illustrated as underscore):
```javascript
/* eslint no-trailing-spaces: 1 */
// trailing space _
```
ESLint produces the same warning as for lines of code: `Trailing spaces not allowed  no-trailing-spaces`

I believe this rule should be relaxed for comments, or at least there should be an option to configure this.

**Comment by gyandeeps** [comment]
Closing this as this is old and no one showed any interest.
Feel free to reopen if interested.

**Comment by moll** [comment]
Was showing interest in the form of pull requests or interesting in having something like this? If the latter, I'm all thumbs up!

**Comment by mik-jozef** [comment]
I'd also welcome this. Sometimes I do
```
/**
 * A long comment with multiple paragraphs.
 *     <- empty line, trailing space
 * Blabla
 */
exports.myWonderfulMethod(needs, long, documentation) { ...
```

**Comment by monolithed** [comment]
+1. 

```js
	/**
	 * Возвращает строку с куками в объектной форме
	 *
	 * @static
	 * @param {string} domain — некоторые драйвера (например, PhantomJS) 
	 *                          требуют это поле
	 * @param {string} cookies
	 * @returns {Array.<Object>}
	 */
	static JSON (domain, cookies) {}
```

I want to format my text annotations without any warnings

**Comment by mavrick** [comment]
+1

**Comment by arxpoetica** [comment]
This is a real thing, not just a want. Editors like Atom put a trailing space when commenting out (`cmd+/`) blank lines that are sandwiched between other lines. Example:

Before:
```
var foo = 'bar';

console.log(`no way ${foo}`);
```

After:
```
// var foo = 'bar';
// 
// console.log(`no way ${foo}`);
```

Note the extra added space by atom after the blank line comment, i.e., `// `. Imagine the tedium now with hundreds of blank lines. Quick comment (`cmd+/`) is no longer quick comment. It's labor intensive comment.

In other words, I have to turn this rule off until this `+1` is fixed.

**Comment by arxpoetica** [comment]
FYI, this is not **done** but I started the ball rolling, if anyone feels like picking up the slack, OR, when I have more downtime I'll finish it off: https://github.com/eslint/eslint/pull/7203
